ui/src/testing/stubs/user-metrics-api-stub.ts

```diff
@@ -10,9 +8,7 @@ export const userMetricsApiStubResources = [
 ];
 export class UserMetricsApiStub extends UserMetricsApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.